dts/bindings/wifi/nordic,nrf7002.yaml

```diff
@@ -21,24 +21,8 @@ properties:
         required: true
         description: |
             GPIO of the SOC controlling the HOST IRQ pin of the nRF7002
-    coex-req-gpios:
-        type: phandle-array
-        required: true
-        description: |
-            GPIO of the SOC controlling SR Coexistence REQ pin of the nRF7002
-    coex-status0-gpios:
-        type: phandle-array
-        required: true
-        description: |
-            GPIO of the SOC controlling SR Coexistence STATUS pin of the nRF7002
-    coex-grant-gpios:
-        type: phandle-array
-        required: true
-        description: |
-            GPIO of the SOC controlling SR Coexistence GRANT pin of the nRF7002
-    swctrl1-gpios:
-        type: phandle-array
-        required: true
+    coex:
+        type: phandle
+        required: false
         description: |
-            GPIO of the SOC controlling the Priority (STATUS1) pin (in 4-wire
-            coex case) of the nRF7002
+            Reference to the nordic,nrf700x-coex coexistence interface.
```

Comment:
At the moment it is not used anywhere, I've just added it for coherency - I thought it would be better to show, that this coex interface is a part of the WiFi interface.

But probably you are right, this is not really needed and Coex is at the moment completely separate.

Reply:
Ok, deleted and cleaned up documenation